sync/class.jetpack-sync-module-updates.php

```diff
@@ -28,6 +31,12 @@ public function init_listeners( $callable ) {
 		), 10, 2 );
 
 		add_action( 'automatic_updates_complete', $callable );
+
+		// Send data when update completes
+		add_action( '_core_updated_successfully', array( $this, 'update_core' ) );
+		add_action( 'jetpack_sync_update_core_successful', $callable, 10, 2 );
```

Comment:
I think we could call this one `jetpack_core_updated_successfully` to match the core one.